### DISCUSSION TITLE
Github student package not uploading documents

### DISCUSSION THREAD

**Discussion by Joabutt** [community/community#101938] [GitHub Education]
### Hi everyone,

Hello, i have this error where I am uploading my documents for my github student verification, but it stays stuck at proccessing request, and the console says that 413 (Payload Too Large), but my image is within the 10mb limit, not sure why this is happening.

<img width="467" alt="Screenshot 2024-01-25 at 1 03 56 PM" src="https://github.com/community/community/assets/89288499/b0f1ca8d-26cb-40cd-adcd-5fc98cf93e34">
<img width="579" alt="Screenshot 2024-01-25 at 1 04 09 PM" src="https://github.com/community/community/assets/89288499/2d05447e-0bfb-4a01-9c15-a3fa45f67c71">

**Comment by amankr1469** [comment]
You should try to resize your screenshots if you submitted any.

**Comment by Joabutt** [comment]
EDIT: I have found the solution! For some reason, uploading an image causes this error, but if you take it using your webcam, it clears! Not sure why this happens though.
